cmd/localstack/main.go

```diff
@@ -33,6 +34,8 @@ func InitLsOpts() *LsOpts {
 		// optional with default
 		InteropPort:     GetenvWithDefault("LOCALSTACK_INTEROP_PORT", "9563"),
 		InitTracingPort: GetenvWithDefault("LOCALSTACK_RUNTIME_TRACING_PORT", "9564"),
+		// optional or empty
+		CodeDownloadUrl: os.Getenv("CODE_ARCHIVE_DOWNLOAD_URL"),
```

Comment:
I feel like we should still namespace this to avoid any (admittedly unlikely) conflicts.

Reply:
Agree!